apps/web/src/pages/settings/organization/OrganizationPage.tsx

```diff
@@ -0,0 +1,28 @@
+import { useAuthContext } from '@novu/shared-web';
+import { CONTEXT_PATH } from '../../../config';
+import { Stack } from '../../../styled-system/jsx';
+import { SettingsPageContainer } from '../SettingsPageContainer';
+import { OrganizationLogo } from './OrganizationLogo';
+import { OrganizationName } from './OrganizationName';
+
+const PAGE_TITLE = 'Organization profile';
+
+/**
+ *TODO: Once organization logo api is implemented
+ * update this to use the org logo
+ */
+const LOGO = CONTEXT_PATH + '/static/images/novu.svg';
+
+export function OrganizationPage() {
+  const { currentOrganization } = useAuthContext();
+
+  return (
+    <SettingsPageContainer title={PAGE_TITLE}>
+      <Stack gap={150} direction="row" h={72} align="flex-end">
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Preference for explicit units, and using `rem` where the size is a multiple of our base unit (0.25rem = 4px)‏
```suggestion
      <Stack gap={150} direction="row" h={'4.5rem'} align="flex-end">
```